cli/tests/pcluster/templates/test_cdk_builder.py

```diff
@@ -42,7 +43,30 @@ def dummy_cluster():
     queue_networking = QueueNetworking(subnet_ids=["test"])
     queues = [SlurmQueue(name="test", networking=queue_networking, compute_resources=compute_resources)]
     scheduling = SlurmScheduling(queues=queues)
-    return SlurmCluster(image=image, head_node=head_node, scheduling=scheduling)
+    # shared storage
+    shared_storage: List[SharedStorage] = []
+    shared_storage.append(
+        SharedFsx(
```

Comment:
I'd like to use a function to generate the dummy fsx, so we can use different parameters to test template generation in future.